src/functions/exp.js

```diff
@@ -1,3 +1,13 @@
+/**
+ * Calculates the exponent of a number. For arrays, the function will be applied index-wise to each element.
```

Comment:
This seems like a better description: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/exp